.circleci/config.yml

```diff
@@ -313,6 +369,51 @@ jobs:
     steps:
       - clair-scan:
           image_name: "astronomerinc/ap-airflow:<< parameters.airflow_version >>-<< parameters.distribution_name >>"
+  scan-trivy:
+    docker:
+      - image: docker:18.09-git
+    description: "Trivy: Vulnerability scanner a Docker image"
+    parameters:
+      airflow_version:
+        description: "The Airflow version, for example '1.10.5'"
+        type: string
+      distribution_name:
+        description: "The base distribution of the container"
+        type: string
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Load archived Docker image
+          command: docker load -i /tmp/workspace/saved-images/astronomerinc/ap-airflow:<< parameters.airflow_version >>-<< parameters.distribution_name >>.tar
+      - run:
+          name: Install trivy
+          command: |
+            apk add --update-cache --upgrade curl rpm
+            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
+      - restore_cache:
+          keys:
+            - trivy-cache
+      - run:
+          name: Scan the local image with trivy
+          command: |
+            cat \<<EOF > .trivyignore
```

Comment:
What did we do for Clair ignore file? My preference would be to have this directly as a file in the repo

Reply:
fixed.